pages/builds/troubleshooting-builds.md.erb

```diff
@@ -0,0 +1,11 @@
+# Troubleshooting
+
+<%= toc %>
+
+## The build I just created is running, but isn't showing up in the My Builds menu
+
+The most common reason for a build to not be included in your My Builds menu is that the email address isn't linked to your account. Make sure that the email attached to the git user that you used to trigger the build is linked to your Buildkite account. Without this link, we won't know that the most recent build is yours!
+
+## The build that was created from my Pull Request failed immediately after starting
+
+Check to make sure that the email address attached to your git user account has permission to create builds in your Buildkite organisation. When a build is created by a webhook from somewhere like Github, the first thing Buildkite will do is check the permissions of the author of the most recent commit. If that author's email isn't linked to the Buildkite organisation you're creating a build under, the build will fail. 
```

Comment:
Ah, I think this has changed since then. It might have been related to teams. Or perhaps to build triggering… but I think we're planning on changing that too! So for now I'd just assume that builds won't failed because of unmatched emails.